pwa/src/templates/organizationDetail/OrganizationDetailTemplate.module.css

```diff
@@ -84,6 +84,7 @@
 }
 
 .tabAmountBadge {
+  --utrecht-badge-counter-padding-inline: 12px;
```

Comment:
See the other comment about this padding.

Seeing this second implementation: I'm thinking we should probably just add this padding to the `component-overrides`, and set the actual values in the theme files.